### PR TITLE
Add exit/quit dot commands

### LINKED ISSUE #38: "exit" should be a .command
"exit" should be a .command

### PR DESCRIPTION
**Pull request by jamesls**
Closes #38.

### CODE REVIEW

File: awsshell/app.py [hunk 1]
```diff
@@ -204,12 +212,12 @@ def run(self):
                 self.save_config()
                 break
             else:
-                if text.strip() in ['quit', 'exit']:
-                    break
                 if text.startswith('.'):
                     # These are special commands.  The only one supported for
                     # now is .edit.
```
Comment (donnemartin) [inline awsshell/app.py]: Should this comment be updated?

**Comment by donnemartin** [comment]
Looks good, just a minor comment.  :shipit:

File: awsshell/app.py (hunk 1 above)
Comment (jamesls) [inline awsshell/app.py]: Yep, I'll update this comment.

**Comment by artburkart** [comment]
Were these removed?

**Comment by donnemartin** [comment]
I just tried a few dot commands, they work on my end.  Are you prefixing them with a `.` ?

Examples:

    aws> .cd [dir]
    aws> .exit

**Comment by artburkart** [comment]
Oh, I'm sorry! I was running an older version (0.0.1) where `exit` and `quit` worked and then they disappeared when I upgraded. I was looking at this commit and was very confused that they had been removed and I've never heard of a 'dot command', so I didn't know what it actually meant.

Now I understand. Everything works fine, cheers.

**Comment by donnemartin** [comment]
No problem :)

**Pull request merged**